lib/html_factory.rb

```diff
@@ -760,4 +739,33 @@ def content(content_lists, callouts, sponsors)
     DOC
     string << content_string
   end
+
+  def section_title(html_class, title, has_hr)
+    if has_hr
```

Comment:
is "Join Us" the only one with no hr? If so, it might be more sense to have a default value of true.